config/spryk/spryks/Spryker/Glue/ApiApplication/Controller/AddGlueApiApplication.yml

```diff
@@ -165,16 +165,16 @@ preSpryks:
               withInterface:
                   value: false
 
-    # Add GlueBackendApiApplicationDependencyProvider
+    # Add GlueApiApplicationDependencyProvider
     - AddGlueDependencyProvider:
           arguments:
               module:
-                  value: "GlueBackendApiApplication"
-    # Methods for the GlueBackendApiApplicationDependencyProvider
+                  value: "GlueApiApplication"
```

Comment:
Hm, what is this value affecting? If this is the DP to which we are going to add the resources/routes then it should be either `GlueBackendApiApplication` or `GlueStorefrontApiApplication`.

Reply:
I have this question/comment to all the changes in this file.

Reply:
That was wrong change but now I fixed it with correct generation so it can be only `Glue{{ applicationType}}ApiApplication` .